Microsoft.Toolkit.Uwp.UI.Media/Geometry/CanvasDrawingSessionExtensions.cs

```diff
@@ -9,7 +9,7 @@
 using Windows.Foundation;
 using Windows.UI;
 
-namespace Microsoft.Toolkit.Uwp.UI.Media.Geometry
+namespace Microsoft.Toolkit.Uwp.UI.Media
```

Comment:
We shouldn't change namespaces for existing classes as we're in a point release. Better to file an issue for an 8.0 major release to clean-up later and link to it here.

Reply:
Ok, I will change them back to their original namespace. 
I have raised #4124 for this issue.

Reply:
Reverted to original namespace.